Changes

```diff
@@ -483,12 +483,12 @@ OCaml 5.4.0
   Xavier Leroy, Miod Vallat, Gabriel Scherer and Stephen Dolan, review by Miod
   Vallat, Vincent Laviron and Xavier Leroy)
 
-* #13050: Use '$' instead of '.' to separate module names in symbol names.
-  This changes mangling of OCaml identifiers from
+* #13050, #14104, #?????: Use '$' instead of '.' to separate module names
+  in symbol names on Apple and non-unix operating systems.
```

Comment:
I think being more explicit about which platforms are affected would be a bit nicer:
```suggestion
  in symbol names on macOS and Windows (including the Cygwin backend).
```